scripts/components/Main.js

```diff
@@ -6,6 +6,7 @@ import ModelViewer from './ModelViewer/ModelViewer';
 import Dialog from './Dialog/Dialog';
 import InteractionContent from './Dialog/InteractionContent';
 import ToolBar from './Toolbar/Toolbar';
+import AudioButton from './HUD/Buttons/AudioButton.js';
```

Comment:
Dunno :-) A linter would complain at least. The issue would pass certain tests, howeverm because AudioButton might as well be defined globally. But someone trying this out should have noticed this.